aksel.nav.no/website/pages/templates/soknad-introside-aap/soknad-introside-aap.tsx

```diff
@@ -0,0 +1,314 @@
+    styles.href = `${MILJO_URL}/css/client.css`;
+    styles.rel = "stylesheet";
+    document.head.appendChild(styles);
+
+    return () => {
+      document.body.removeChild(script);
+      document.head.removeChild(styles);
+    };
+  }, []);
+}
+
+const ApplicationPictogram = ({ title, ...props }) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="72"
+    height="72"
+    viewBox="0 0 72 72"
+    fill="none"
+    style={{ flexShrink: 0 }}
+    aria-hidden
+    {...props}
+  >
+    <title>{title}</title>
+    <rect x="23.25" y="22.5" width="26.25" height="9" fill="#CCE2F0" />
+    <rect x="23.25" y="36.75" width="26.25" height="9" fill="#CCE2F0" />
+    <circle cx="36.75" cy="34.5" r="21" fill="#CCE2F0" />
+    <path
+      d="M23.7672 5.508L30.1202 11.8434M1.5 33.75H34.5M26.4706 2.81211L10.5882 18.6506L9 26.5699L16.9412 24.986L32.8235 9.14751C34.5778 7.39804 34.5778 4.56158 32.8235 2.81211C31.0692 1.06263 28.2249 1.06263 26.4706 2.81211Z"
+      stroke="#23262A"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M56.25 44.25L63.75 44.25"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M56.25 52.5L63.75 52.5"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M56.25 60.75L63.75 60.75"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M48 52.5L51 52.5"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M48 52.5L51 52.5"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M48 44.25L51 44.25"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M48 44.25L51 44.25"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M48 60.75L51 60.75"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M48 60.75L51 60.75"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <rect
+      x="41.25"
+      y="33"
+      width="29.25"
+      height="37.5"
+      stroke="#262626"
+      strokeWidth="2.25"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+
+// EXAMPLES DO NOT INCLUDE CONTENT BELOW THIS LINE
+export default Example;
+
+/* Storybook story */
+export const Demo = {
+  render: Example,
+};
+
+export const args = {
+  index: 1,
+  title: "Introside AAP",
```

Comment:
I think `title` is redundant when there's only example in the folder.

Reply:
Hva med `index`? Jeg kjenner ikke nok til hvordan dette henger sammen, så har bare kopiert fra andre.

Reply:
Usikker på om den _må_ være med, men den _pleier_ alltid å være med.